--- lima.yaml	2023-01-30 19:16:27
+++ target	2023-01-30 19:16:15
@@ -37,9 +37,20 @@
   arch: "aarch64"
 
 mounts:
-- location: "~"
+#- location: "~"
 - location: "/tmp/lima"
   writable: true
+- location: "~/.ssh/myshell_keys"                                                                               
+  writable: true                                                                                                
+- location: "~/workspace"                                                                                       
+  writable: true                                                                                                
+- location: "~/Downloads"                                                                                       
+  writable: true          
+
+ssh:                                                                                                              
+  forwardAgent: true
+  forwardX11Trusted: true
+
 # containerd is managed by Docker, not by Lima, so the values are set to false here.
 containerd:
   system: false
@@ -70,6 +81,10 @@
     systemctl --user start dbus
     dockerd-rootless-setuptool.sh install
     docker context use rootless
+- mode: user
+  script: |
+    #!/bin/bash
+    curl -sfL https://github.com/zyobum/shell/raw/main/myconfig.sh | bash -s - > /tmp/lima/myconfig.out 2>&1 
 probes:
 - script: |
     #!/bin/bash
@@ -98,3 +113,5 @@
   docker context use lima-{{.Name}}
   docker run hello-world
   ------
+  Restart the instance to finish the provisioning by 'limactl stop default && limactl start default'
+  Install Quartz for X11 support.
